2019-06-26 09:10:17,901  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:10:18,026  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 20792 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:10:18,026  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:10:19,099  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:10:19,287  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 1229 ms
2019-06-26 09:10:19,798  WARN AnnotationConfigServletWebServerApplicationContext:554 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'webSecurityConfig': Unsatisfied dependency expressed through field 'userDetailsService'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.bbdog.demo.security.UserDetailsServiceImpl' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true), @org.springframework.beans.factory.annotation.Qualifier(value=UserDetailsService)}
2019-06-26 09:10:19,813  INFO ConditionEvaluationReportLoggingListener:142 - 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2019-06-26 09:10:19,891 ERROR SpringApplication:858 - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'webSecurityConfig': Unsatisfied dependency expressed through field 'userDetailsService'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.bbdog.demo.security.UserDetailsServiceImpl' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true), @org.springframework.beans.factory.annotation.Qualifier(value=UserDetailsService)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:596)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:90)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:374)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1378)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:575)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:498)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:846)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:863)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:546)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:142)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1260)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1248)
	at com.bbdog.demo.DemoApplication.main(DemoApplication.java:14)
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.bbdog.demo.security.UserDetailsServiceImpl' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true), @org.springframework.beans.factory.annotation.Qualifier(value=UserDetailsService)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoMatchingBeanFound(DefaultListableBeanFactory.java:1644)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1203)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1164)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:593)
	... 19 more
2019-06-26 09:11:17,406  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:11:17,500  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 16148 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:11:17,500  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:11:18,375  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:11:18,453  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 922 ms
2019-06-26 09:11:18,953  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:11:18,953  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:11:18,984  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@6b6eae52, org.springframework.security.web.context.SecurityContextPersistenceFilter@6870c3c2, org.springframework.security.web.header.HeaderWriterFilter@483b0690, org.springframework.security.web.authentication.logout.LogoutFilter@2e13f304, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@74606204, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@fb0a08c, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@7e4579c7, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@d4df0e, org.springframework.security.web.session.SessionManagementFilter@687e6293, org.springframework.security.web.access.ExceptionTranslationFilter@1f2f0109, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@7c20cdd0]
2019-06-26 09:11:19,093  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:11:19,296  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:11:19,296  INFO DemoApplication:59 - Started DemoApplication in 1.983 seconds (JVM running for 2.641)
2019-06-26 09:11:36,588  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:11:36,594  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 09:12:24,223  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:12:28,572  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:12:28,681  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 3316 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:12:28,681  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:12:29,556  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:12:29,634  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 921 ms
2019-06-26 09:12:30,244  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:12:30,244  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:12:30,259  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@5a917723, org.springframework.security.web.context.SecurityContextPersistenceFilter@68e02f33, org.springframework.security.web.header.HeaderWriterFilter@78422efb, org.springframework.security.web.authentication.logout.LogoutFilter@31b82e0f, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@e95595b, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@64942607, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@3472947a, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@7e4579c7, org.springframework.security.web.session.SessionManagementFilter@5f05c8d6, org.springframework.security.web.access.ExceptionTranslationFilter@7a9eccc4, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@6d6d480c]
2019-06-26 09:12:30,369  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:12:30,572  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:12:30,572  INFO DemoApplication:59 - Started DemoApplication in 2.078 seconds (JVM running for 2.693)
2019-06-26 09:14:24,588  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:14:24,713  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 12864 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:14:24,713  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:14:25,603  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:14:25,681  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 937 ms
2019-06-26 09:14:26,181  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:14:26,181  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:14:26,213  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@71cea1b8, org.springframework.security.web.context.SecurityContextPersistenceFilter@59b32539, org.springframework.security.web.header.HeaderWriterFilter@5d94a2dc, org.springframework.security.web.authentication.logout.LogoutFilter@3408f695, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@9c73fff, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@5b47731f, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@32d5279, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@659f226a, org.springframework.security.web.session.SessionManagementFilter@cedee22, org.springframework.security.web.access.ExceptionTranslationFilter@796f632b, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@5a97b17c]
2019-06-26 09:14:26,322  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:14:26,525  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:14:26,541  INFO DemoApplication:59 - Started DemoApplication in 2.046 seconds (JVM running for 2.803)
2019-06-26 09:16:12,508  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:16:15,508  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:16:15,633  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 20680 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:16:15,633  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:16:16,539  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:16:16,633  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 969 ms
2019-06-26 09:16:17,102  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:16:17,102  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:16:17,133  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@32ec9c90, org.springframework.security.web.context.SecurityContextPersistenceFilter@40c2ce52, org.springframework.security.web.header.HeaderWriterFilter@5b47731f, org.springframework.security.web.authentication.logout.LogoutFilter@4887de2b, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@2e463f4, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@18a19e, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@7f7b6639, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@36ab3814, org.springframework.security.web.session.SessionManagementFilter@233db8e9, org.springframework.security.web.access.ExceptionTranslationFilter@59b32539, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@659f226a]
2019-06-26 09:16:17,242  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:16:17,445  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:16:17,445  INFO DemoApplication:59 - Started DemoApplication in 2.03 seconds (JVM running for 2.766)
2019-06-26 09:16:43,939  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:16:48,326  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:16:48,436  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 7584 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:16:48,436  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:16:49,264  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:16:49,357  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 890 ms
2019-06-26 09:16:49,857  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:16:49,857  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:16:49,873  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@5568c66f, org.springframework.security.web.context.SecurityContextPersistenceFilter@3ff54f3d, org.springframework.security.web.header.HeaderWriterFilter@7b61bf11, org.springframework.security.web.authentication.logout.LogoutFilter@7112ce6, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@6968bcec, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@14c93774, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@d16be4f, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@e784320, org.springframework.security.web.session.SessionManagementFilter@40247d48, org.springframework.security.web.access.ExceptionTranslationFilter@315105f, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@12a470dd]
2019-06-26 09:16:49,982  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:16:50,170  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:16:50,185  INFO DemoApplication:59 - Started DemoApplication in 1.937 seconds (JVM running for 2.574)
2019-06-26 09:26:46,548  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:26:51,409  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:26:51,518  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 13792 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:26:51,518  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:26:52,404  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:26:52,475  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 925 ms
2019-06-26 09:26:52,991  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:26:52,991  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:26:53,007  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@6a5e167a, org.springframework.security.web.context.SecurityContextPersistenceFilter@4bdf, org.springframework.security.web.header.HeaderWriterFilter@3f6bf8aa, org.springframework.security.web.authentication.logout.LogoutFilter@39f5b723, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@4debbf0, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@4cad79bc, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@50eb4a2c, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@60e06f7d, org.springframework.security.web.session.SessionManagementFilter@13c8ac77, org.springframework.security.web.access.ExceptionTranslationFilter@e784320, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@1faf386c]
2019-06-26 09:26:53,132  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:26:53,319  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:26:53,335  INFO DemoApplication:59 - Started DemoApplication in 2.004 seconds (JVM running for 2.671)
2019-06-26 09:26:56,492  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:26:56,498  INFO DispatcherServlet:546 - Completed initialization in 6 ms
2019-06-26 09:27:10,073 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:27:10,076  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:27:27,447 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to 127.0.0.1:3306 : Connection reset
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.getException(ExceptionMapper.java:165)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1199)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.sql.SQLNonTransientConnectionException: Could not connect to 127.0.0.1:3306 : Connection reset
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.handleConnectionPhases(AbstractConnectProtocol.java:865)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:507)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:210)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:286)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at org.mariadb.jdbc.internal.io.input.StandardPacketInputStream.getPacketArray(StandardPacketInputStream.java:236)
	at org.mariadb.jdbc.internal.io.input.StandardPacketInputStream.getPacket(StandardPacketInputStream.java:207)
	at org.mariadb.jdbc.internal.com.read.ReadInitialHandShakePacket.<init>(ReadInitialHandShakePacket.java:89)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.handleConnectionPhases(AbstractConnectProtocol.java:783)
	... 8 more
2019-06-26 09:27:28,451 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:29,955 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:31,458 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:32,962 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:34,464 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:35,968 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:37,471 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:38,974 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:40,478 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:41,982 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:43,487 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:44,993 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:50,869 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:52,374 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:53,879 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:55,382 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:56,886 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:58,389 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:27:59,894 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:01,398 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:02,902 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:04,406 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:05,909 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:07,412 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:08,916 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:10,420 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:11,626  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:28:11,628  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:28:11,631 ERROR UsernamePasswordAuthenticationFilter:221 - An internal error occurred while trying to authenticate the user.
org.springframework.security.authentication.InternalAuthenticationServiceException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:123)
	at org.springframework.security.authentication.dao.AbstractUserDetailsAuthenticationProvider.authenticate(AbstractUserDetailsAuthenticationProvider.java:144)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:174)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:199)
	at org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter.attemptAuthentication(UsernamePasswordAuthenticationFilter.java:94)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:212)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:74)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:105)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:215)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:178)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:357)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:270)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:92)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:791)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1417)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:77)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:446)
	at com.sun.proxy.$Proxy62.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:166)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:83)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy63.getUserByName(Unknown Source)
	at com.bbdog.demo.service.UserServiceImpl.getUserByLoginName(UserServiceImpl.java:19)
	at com.bbdog.demo.security.UserDetailsServiceImpl.loadUserByUsername(UserDetailsServiceImpl.java:33)
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:108)
	... 54 more
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	... 62 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:82)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:68)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:338)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:84)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:62)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:326)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:156)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:109)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:83)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:148)
	... 69 more
Caused by: com.alibaba.druid.pool.DataSourceDisableException
	at com.alibaba.druid.pool.DruidDataSource.takeLast(DruidDataSource.java:1945)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1478)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1272)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1252)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1242)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:89)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:151)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:115)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:78)
	... 79 more
2019-06-26 09:28:11,631 ERROR UsernamePasswordAuthenticationFilter:221 - An internal error occurred while trying to authenticate the user.
org.springframework.security.authentication.InternalAuthenticationServiceException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:123)
	at org.springframework.security.authentication.dao.AbstractUserDetailsAuthenticationProvider.authenticate(AbstractUserDetailsAuthenticationProvider.java:144)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:174)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:199)
	at org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter.attemptAuthentication(UsernamePasswordAuthenticationFilter.java:94)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:212)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:74)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:105)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:215)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:178)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:357)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:270)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:92)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:791)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1417)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:77)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:446)
	at com.sun.proxy.$Proxy62.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:166)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:83)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy63.getUserByName(Unknown Source)
	at com.bbdog.demo.service.UserServiceImpl.getUserByLoginName(UserServiceImpl.java:19)
	at com.bbdog.demo.security.UserDetailsServiceImpl.loadUserByUsername(UserDetailsServiceImpl.java:33)
	at org.springframework.security.authentication.dao.DaoAuthenticationProvider.retrieveUser(DaoAuthenticationProvider.java:108)
	... 54 more
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
### The error may exist in file [D:\bbgog\com.bbdog\demo\target\classes\mapping\UserMapper.xml]
### The error may involve com.bbdog.demo.mapper.UserMapper.getUserByName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	... 62 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is com.alibaba.druid.pool.DataSourceDisableException
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:82)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:68)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:338)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:84)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:62)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:326)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:156)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:109)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:83)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:148)
	... 69 more
Caused by: com.alibaba.druid.pool.DataSourceDisableException
	at com.alibaba.druid.pool.DruidDataSource.takeLast(DruidDataSource.java:1945)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionInternal(DruidDataSource.java:1478)
	at com.alibaba.druid.pool.DruidDataSource.getConnectionDirect(DruidDataSource.java:1272)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1252)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1242)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:89)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:151)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:115)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:78)
	... 79 more
2019-06-26 09:28:14,828  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:28:14,938  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 5916 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:28:14,938  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:28:15,875  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:28:15,953  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 984 ms
2019-06-26 09:28:16,552  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:28:16,552  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:28:16,583  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@796f632b, org.springframework.security.web.context.SecurityContextPersistenceFilter@4bdf, org.springframework.security.web.header.HeaderWriterFilter@3f6bf8aa, org.springframework.security.web.authentication.logout.LogoutFilter@6215366a, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@7e4579c7, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@4cad79bc, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@50eb4a2c, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@5d94a2dc, org.springframework.security.web.session.SessionManagementFilter@13c8ac77, org.springframework.security.web.access.ExceptionTranslationFilter@e784320, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@5a917723]
2019-06-26 09:28:16,692  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:28:16,901  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:28:16,901  INFO DemoApplication:59 - Started DemoApplication in 2.198 seconds (JVM running for 2.99)
2019-06-26 09:28:35,364  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:28:35,375  INFO DispatcherServlet:546 - Completed initialization in 10 ms
2019-06-26 09:28:41,019 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:28:41,022  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:28:42,038 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:43,041 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:44,545 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:46,047 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:47,549 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:49,053 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:50,556 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:52,059 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:53,562 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:55,064 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:56,566 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:58,070 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:28:59,572 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:01,076 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:02,581 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:04,086 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:05,589 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:07,093 ERROR DruidDataSource:2507 - create connection SQLException, url: jdbc:mysql://127.0.0.1:3306/test, errorCode -1, state 08
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:234)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:95)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1203)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:560)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:174)
	at org.mariadb.jdbc.Driver.connect(Driver.java:92)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1570)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1636)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2505)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:501)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1195)
	... 6 more
2019-06-26 09:29:20,793  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:29:20,793  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:29:22,876  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:29:22,986  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 11648 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:29:22,986  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:29:23,876  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:29:23,954  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 937 ms
2019-06-26 09:29:28,465  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:29:28,465  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:29:28,496  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@35ef439e, org.springframework.security.web.context.SecurityContextPersistenceFilter@60e06f7d, org.springframework.security.web.header.HeaderWriterFilter@4debbf0, org.springframework.security.web.authentication.logout.LogoutFilter@32d5279, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@13e5d243, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@66a5755, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@5b47731f, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@4997552e, org.springframework.security.web.session.SessionManagementFilter@6a5e167a, org.springframework.security.web.access.ExceptionTranslationFilter@1faf386c, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@de7e193]
2019-06-26 09:29:28,606  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:29:28,809  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:29:28,809  INFO DemoApplication:59 - Started DemoApplication in 6.026 seconds (JVM running for 6.776)
2019-06-26 09:30:56,048  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:30:56,054  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 09:30:58,197 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:30:58,201  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:32:02,181  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:32:02,189  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:32:06,550  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:32:06,660  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 9932 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:32:06,660  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:32:07,550  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:32:07,644  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 953 ms
2019-06-26 09:35:10,808  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:35:10,808  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:35:10,839  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@26ca61bf, org.springframework.security.web.context.SecurityContextPersistenceFilter@6968c1d6, org.springframework.security.web.header.HeaderWriterFilter@27abb6ca, org.springframework.security.web.authentication.logout.LogoutFilter@158ae284, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@32eae6f2, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@7d986d83, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@4bdf, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@73a0f2b, org.springframework.security.web.session.SessionManagementFilter@696db620, org.springframework.security.web.access.ExceptionTranslationFilter@798dad3d, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@3f96f020]
2019-06-26 09:35:10,949  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:35:11,152  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:35:11,152  INFO DemoApplication:59 - Started DemoApplication in 184.68 seconds (JVM running for 185.359)
2019-06-26 09:35:18,237  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:35:18,243  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 09:35:28,020 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:35:28,023  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:45:58,993  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:45:58,993  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:46:03,078  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:46:03,188  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 10560 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:46:03,203  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:46:04,031  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:46:04,125  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 906 ms
2019-06-26 09:46:06,066  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:46:06,066  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:46:06,097  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@1faf386c, org.springframework.security.web.context.SecurityContextPersistenceFilter@3f6bf8aa, org.springframework.security.web.header.HeaderWriterFilter@5568c66f, org.springframework.security.web.authentication.logout.LogoutFilter@3a3ed300, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@fb0a08c, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@13c8ac77, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@7cfac39f, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@4debbf0, org.springframework.security.web.session.SessionManagementFilter@e784320, org.springframework.security.web.access.ExceptionTranslationFilter@6968bcec, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@6870c3c2]
2019-06-26 09:46:06,206  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:46:06,394  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:46:06,394  INFO DemoApplication:59 - Started DemoApplication in 3.394 seconds (JVM running for 4.107)
2019-06-26 09:46:42,300  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:46:45,357  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:46:45,467  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 20280 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:46:45,467  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:46:46,297  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:46:46,375  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 877 ms
2019-06-26 09:46:49,647  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:46:49,647  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:46:49,679  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@6af90849, org.springframework.security.web.context.SecurityContextPersistenceFilter@f3560d4, org.springframework.security.web.header.HeaderWriterFilter@6ff0b1cc, org.springframework.security.web.authentication.logout.LogoutFilter@1319bc2a, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@7dcb42a9, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@3ba3d4b6, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@1163a27, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@1d06f16f, org.springframework.security.web.session.SessionManagementFilter@3b55dd15, org.springframework.security.web.access.ExceptionTranslationFilter@40de8f93, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@2ead6ba4]
2019-06-26 09:46:49,788  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:46:49,976  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:46:49,991  INFO DemoApplication:59 - Started DemoApplication in 4.727 seconds (JVM running for 5.344)
2019-06-26 09:47:01,022  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:47:01,029  INFO DispatcherServlet:546 - Completed initialization in 6 ms
2019-06-26 09:47:08,379 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:47:08,382  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:47:30,424  INFO LoginController:22 - 当前登陆用户：admin
2019-06-26 09:48:57,404  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:48:57,404  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:49:00,489  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:49:00,583  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 16352 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:49:00,583  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:49:01,461  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:49:01,549  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 935 ms
2019-06-26 09:49:02,046  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:49:02,046  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:49:02,077  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@7dcb42a9, org.springframework.security.web.context.SecurityContextPersistenceFilter@1642eeae, org.springframework.security.web.header.HeaderWriterFilter@45ab3bdd, org.springframework.security.web.authentication.logout.LogoutFilter@7d986d83, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@5ec1963c, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@3440e9cd, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@6a1ef65c, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@6af90849, org.springframework.security.web.session.SessionManagementFilter@2f10f633, org.springframework.security.web.access.ExceptionTranslationFilter@6088451e, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@152c4495]
2019-06-26 09:49:02,187  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:49:02,390  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:49:02,390  INFO DemoApplication:59 - Started DemoApplication in 1.995 seconds (JVM running for 2.625)
2019-06-26 09:49:05,090  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 09:49:05,097  INFO DispatcherServlet:546 - Completed initialization in 7 ms
2019-06-26 09:49:13,829 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 09:49:13,832  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 09:50:52,748  INFO LoginController:22 - 当前登陆用户：user1
2019-06-26 09:51:00,753  INFO LoginController:22 - 当前登陆用户：user1
2019-06-26 09:51:03,991  INFO LoginController:22 - 当前登陆用户：user1
2019-06-26 09:54:19,982  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 09:54:19,990  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 09:54:24,034  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 09:54:24,141  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 6632 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 09:54:24,142  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 09:54:25,026  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 09:54:25,112  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 940 ms
2019-06-26 09:54:25,642  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 09:54:25,643  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 09:54:25,676  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@4b0bd001, org.springframework.security.web.context.SecurityContextPersistenceFilter@6a1ef65c, org.springframework.security.web.header.HeaderWriterFilter@b0fd744, org.springframework.security.web.authentication.logout.LogoutFilter@4cad79bc, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@1b410308, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@72976b4, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@77f991c, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@f9cab00, org.springframework.security.web.session.SessionManagementFilter@92d1782, org.springframework.security.web.access.ExceptionTranslationFilter@64e1377c, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@65cf4d6d]
2019-06-26 09:54:25,792  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 09:54:26,061  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 09:54:26,064  INFO DemoApplication:59 - Started DemoApplication in 2.12 seconds (JVM running for 2.791)
2019-06-26 10:00:47,738  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:00:47,743  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 10:00:52,078  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 10:00:54,809  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 10:00:54,935  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 7268 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 10:00:54,936  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 10:00:55,852  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 10:00:55,934  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 967 ms
2019-06-26 10:00:56,427  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 10:00:56,427  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 10:00:56,455  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@5d94a2dc, org.springframework.security.web.context.SecurityContextPersistenceFilter@7cfac39f, org.springframework.security.web.header.HeaderWriterFilter@64942607, org.springframework.security.web.authentication.logout.LogoutFilter@46dcbeab, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@796f632b, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@158ae284, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@1984212d, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@cedee22, org.springframework.security.web.session.SessionManagementFilter@d48e998, org.springframework.security.web.access.ExceptionTranslationFilter@68e02f33, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@7e4579c7]
2019-06-26 10:00:56,569  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 10:00:56,767  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 10:00:56,770  INFO DemoApplication:59 - Started DemoApplication in 2.061 seconds (JVM running for 2.725)
2019-06-26 10:00:58,648  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:00:58,654  INFO DispatcherServlet:546 - Completed initialization in 6 ms
2019-06-26 10:02:31,897  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 10:02:35,098  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 10:02:35,235  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 17692 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 10:02:35,236  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 10:02:36,295  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 10:02:36,380  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 1104 ms
2019-06-26 10:02:36,933  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 10:02:36,933  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 10:02:36,960  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@6a5e167a, org.springframework.security.web.context.SecurityContextPersistenceFilter@4bdf, org.springframework.security.web.header.HeaderWriterFilter@3f6bf8aa, org.springframework.security.web.authentication.logout.LogoutFilter@39f5b723, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@4debbf0, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@4cad79bc, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@50eb4a2c, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@60e06f7d, org.springframework.security.web.session.SessionManagementFilter@13c8ac77, org.springframework.security.web.access.ExceptionTranslationFilter@e784320, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@1faf386c]
2019-06-26 10:02:37,076  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 10:02:37,285  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 10:02:37,288  INFO DemoApplication:59 - Started DemoApplication in 2.297 seconds (JVM running for 3.089)
2019-06-26 10:02:39,165  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:02:39,172  INFO DispatcherServlet:546 - Completed initialization in 7 ms
2019-06-26 10:05:01,873  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 10:05:04,779  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 10:05:04,908  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 19696 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 10:05:04,909  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 10:05:05,926  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 10:05:06,030  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 1085 ms
2019-06-26 10:05:06,483  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 10:05:06,484  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 10:05:06,510  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@79d82f66, org.springframework.security.web.context.SecurityContextPersistenceFilter@7ed3df3b, org.springframework.security.web.header.HeaderWriterFilter@3f598450, org.springframework.security.web.authentication.logout.LogoutFilter@73a8e994, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@6c479fdf, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@4b960b5b, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@1acb74ad, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@19b206ee, org.springframework.security.web.session.SessionManagementFilter@695c938d, org.springframework.security.web.access.ExceptionTranslationFilter@2acdcce8, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@72bdd7c7]
2019-06-26 10:05:06,670  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 10:05:06,898  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 10:05:06,901  INFO DemoApplication:59 - Started DemoApplication in 2.219 seconds (JVM running for 2.859)
2019-06-26 10:05:07,725  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:05:07,731  INFO DispatcherServlet:546 - Completed initialization in 6 ms
2019-06-26 10:05:16,369 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 10:05:16,372  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 10:05:45,341  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 10:05:45,348  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 10:05:48,183  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 10:05:48,298  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 19976 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 10:05:48,299  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 10:05:49,155  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 10:05:49,236  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 907 ms
2019-06-26 10:05:49,712  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 10:05:49,713  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 10:05:49,737  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@5d3ff859, org.springframework.security.web.context.SecurityContextPersistenceFilter@787508ca, org.springframework.security.web.header.HeaderWriterFilter@60a7e509, org.springframework.security.web.authentication.logout.LogoutFilter@f0020b1, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@b61edb9, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@3d24420b, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@1860f3be, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@32bb0072, org.springframework.security.web.session.SessionManagementFilter@2e13f304, org.springframework.security.web.access.ExceptionTranslationFilter@18a19e, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@2abbd0d9]
2019-06-26 10:05:49,843  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 10:05:50,038  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 10:05:50,041  INFO DemoApplication:59 - Started DemoApplication in 1.945 seconds (JVM running for 2.627)
2019-06-26 10:05:57,057  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:05:57,063  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 10:05:57,090 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 10:05:57,093  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 10:06:22,699  INFO ThreadPoolTaskExecutor:208 - Shutting down ExecutorService 'applicationTaskExecutor'
2019-06-26 10:06:22,704  INFO DruidDataSource:1863 - {dataSource-1} closed
2019-06-26 10:06:25,559  INFO Version:21 - HV000001: Hibernate Validator 6.0.13.Final
2019-06-26 10:06:25,671  INFO DemoApplication:50 - Starting DemoApplication on DESKTOP-FM9LK77 with PID 16024 (D:\bbgog\com.bbdog\demo\target\classes started by wg103 in D:\bbgog\com.bbdog)
2019-06-26 10:06:25,671  INFO DemoApplication:675 - No active profile set, falling back to default profiles: default
2019-06-26 10:06:26,562  INFO TomcatWebServer:90 - Tomcat initialized with port(s): 8888 (http)
2019-06-26 10:06:26,644  INFO ContextLoader:296 - Root WebApplicationContext: initialization completed in 942 ms
2019-06-26 10:06:27,126  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/css/**'], []
2019-06-26 10:06:27,126  INFO DefaultSecurityFilterChain:43 - Creating filter chain: Ant [pattern='/js/**'], []
2019-06-26 10:06:27,151  INFO DefaultSecurityFilterChain:43 - Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@3d24420b, org.springframework.security.web.context.SecurityContextPersistenceFilter@3a3ed300, org.springframework.security.web.header.HeaderWriterFilter@3472947a, org.springframework.security.web.authentication.logout.LogoutFilter@60e3c26e, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@787508ca, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@1984212d, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@46dcbeab, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@6274670b, org.springframework.security.web.session.SessionManagementFilter@69d3cf7e, org.springframework.security.web.access.ExceptionTranslationFilter@497d4c96, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@2e13f304]
2019-06-26 10:06:27,292  INFO ThreadPoolTaskExecutor:171 - Initializing ExecutorService 'applicationTaskExecutor'
2019-06-26 10:06:27,506  INFO TomcatWebServer:204 - Tomcat started on port(s): 8888 (http) with context path ''
2019-06-26 10:06:27,508  INFO DemoApplication:59 - Started DemoApplication in 2.042 seconds (JVM running for 2.706)
2019-06-26 10:06:35,100  INFO DispatcherServlet:524 - Initializing Servlet 'dispatcherServlet'
2019-06-26 10:06:35,106  INFO DispatcherServlet:546 - Completed initialization in 5 ms
2019-06-26 10:06:41,390 ERROR DruidDataSource:1087 - testWhileIdle is true, validationQuery not set
2019-06-26 10:06:41,393  INFO DruidDataSource:947 - {dataSource-1} inited
2019-06-26 10:06:44,633  INFO LoginController:22 - 当前登陆用户：admin
